doc/autogen/types/sink.rst

```diff
@@ -50,8 +50,8 @@
 
 .. spicy:method:: sink::gap sink gap False void (seq: uint<64>, len: uint<64>)
 
-    Reports a gap in the input stream. *seq* is the sequence number of the
```

Comment:
Have you checked what style is more common across all operators? I thought I had been meaning to use `*XX*` for  parameters, though obviously not consistently. Either way, if we make these changes, should ensure it's what we do elsewhere too.

Reply:
note: the changes below from single-tick to double-tickets are ok, they are just typos accidentally switching to markdown.

Reply:
I missed that, `*par*` seems indeed like the preferred way to refer to parameters. I rolled back the changes on that here.